com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -294,7 +321,11 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
 
     half4 FragFog(Varyings input) : SV_Target
     {
-        float d = LOAD_TEXTURE2D_X(_CameraDepthTexture, input.positionCS.xy).x;
+        #if _RENDER_PASS_ENABLED
+            float d = LOAD_FRAMEBUFFER_INPUT(GBUFFER3, input.positionCS.xy);
```

Comment:
Should this be reference the channel like above:
float d = LOAD_FRAMEBUFFER_INPUT(GBUFFER3, input.positionCS.xy)**.x**;

Reply:
works both ways, but definitely should be using the same way. fixing this!